### DISCUSSION TITLE
Codici Remote IR

### DISCUSSION THREAD

**Discussion by ugoberenice** [indomus/forum#267] [HUB: Home Assistant]
Ciao a tutti.
Ho un problemino in HA ma prima di mettermi e sbattere la testa contro un muro vorrei sapere da qualcuno di voi se è una cosa normale .
Attraverso broadlink manager ho memorizzato codici IR da poter inserire in command 
Il problema che si pone è che , a parte il tasto on/off , tutti gli altri cambiano in continuazione !
Ogni volta che schiaccio il tasto 1 per esempio restituisce un codice differente ed ovviamente una volta inserito in qualche script non funziona più . Come mai? Come faccio a risolvere sto problema ? Thx

**Comment by KC-inDomus** [comment]
che telecomando è

**Reply by ugoberenice** [reply]
Il tv è un Panasonic viera con relativo telecomando di proprietà

**Reply by KC-inDomus** [reply]
hmmm che cambino i codici è estremamente strano, ma se così dici essere così sarà. e se è così, non c'è niente da fare.

**Reply by ugoberenice** [reply]
Allora facendo delle prove ho visto che effettivamente i codici cambiano ma inviandoli sempre tramite broadlink manager funzionano lo stesso . Ho memorizzato due codici differenti per il canale 1 e dal software funzionano entrambi.  Trmite HA però non funziona niente tranne on/off ... non capisco

**Reply by KC-inDomus** [reply]
eh, se l'invio da broadlink va stai sbagliando qualcosa lato HA, ovviamente.

**Reply by indomus** [reply]
Usi i remote di Home Assistant o cos'altro?
